components/support/nimbus-fml/src/backends/swift/templates/FeatureManifestTemplate.swift

```diff
@@ -80,6 +80,22 @@ public class {{ nimbus_object }} : FeatureManifestInterface {
         {%- endfor %}
     }
 
+
+    ///
+    /// Get a list of feature ids where the feature allows co-enrollment.
+    ///
+    public func getCoenrolledFeatureIds(): List<String>? {
+        {%- let features = self.iter_feature_defs() %}
+        {%- if features.is_empty() %}
+        return []
+        {%- let coenrolledFeatures = ArrayList<String> %}
+        {%- for f in features %}
+        {%- if f.allow_coenrollment() %}
+        coenrolledFeatures += features.{{- f.name() -}}
+        {%- endfor %}
+        return coenrolledFeatures
+    }
```

Comment:
I think get one language going before doing the second one. That way you don't need to deal with the same failing tests twice, you get to deal with two different sets of failing tests!